renderer-macos/lib/stubs/BriskView.c

```diff
@@ -1,8 +1,7 @@
-#import "NSView_stubs.h"
-#import <caml/alloc.h>
-#import <caml/callback.h>
-#import <caml/memory.h>
-#import <caml/mlvalues.h>
+#import "BriskCocoa.h"
+
+#define Val_View(v) ((value)(v))
+#define View_val(v) ((__bridge NSView *)(value)(v))
```

Comment:
`__bridge` is probably not what we want 😄. I'd rather remove this until we have some better solution for retaining/releasing views.